src/Tools/dotnet-monitor/Egress/EgressService.cs

```diff
@@ -117,43 +117,30 @@ private async static Task<EgressArtifactSettings> CreateSettings(IEndpointInfo s
             Activity activity = Activity.Current;
             if (null != activity)
             {
-                settings.Metadata.Add(
-                    ActivityMetadataNames.ParentId,
-                    activity.GetParentId());
-                settings.Metadata.Add(
-                    ActivityMetadataNames.SpanId,
-                    activity.GetSpanId());
-                settings.Metadata.Add(
-                    ActivityMetadataNames.TraceId,
-                    activity.GetTraceId());
+                AddMetadata(settings, ActivityMetadataNames.ParentId, activity.GetParentId());
+                AddMetadata(settings, ActivityMetadataNames.SpanId, activity.GetSpanId());
+                AddMetadata(settings, ActivityMetadataNames.TraceId, activity.GetTraceId());
             }
 
             if (null != collectionRuleMetadata)
             {
-                settings.Metadata.Add(
-                    CollectionRuleMetadataNames.CollectionRuleName,
-                    collectionRuleMetadata.CollectionRuleName);
-
-                settings.Metadata.Add(
-                    CollectionRuleMetadataNames.ActionListIndex,
-                    collectionRuleMetadata.ActionListIndex.ToString());
-
-                settings.Metadata.Add(
-                    CollectionRuleMetadataNames.ActionName,
-                    collectionRuleMetadata.ActionName);
+                AddMetadata(settings, CollectionRuleMetadataNames.CollectionRuleName, collectionRuleMetadata.CollectionRuleName);
+                AddMetadata(settings, CollectionRuleMetadataNames.ActionListIndex, collectionRuleMetadata.ActionListIndex.ToString());
```

Comment:
```suggestion
                AddMetadata(settings, CollectionRuleMetadataNames.ActionListIndex, collectionRuleMetadata.ActionListIndex.ToString("D", CultureInfo.InvariantCulture));
```

Reply:
This ensures that the number is formatted without the extra delimiters such as `,`, `.`, and formats invariant instead of the culture of the current user who launched dotnet-monitor.